src/pages/TaskModuleTab.ts

```diff
@@ -0,0 +1,128 @@
+                <script src='https://code.jquery.com/jquery-1.11.3.min.js'></script>
+                </head>
+
+            <body>  
+            <button class="taskModuleButton" id="commandList">Invoke Task Module- Result Tab</button>
+            <br>
+            <label id="taskModuleResult"></label>
+            <br>
+            <br>
+            <button class="taskModuleButton" id="adaptiveCardResultBot">Invoke Task Module- Result Bot</button>
+             <script>
+             var microsoftTeams;
+
+             var  adaptiveCard = {
+                "type": "AdaptiveCard",
+                "body": [
+                    {
+                        "type": "TextBlock",
+                        "separator": true,
+                        "size": "Large",
+                        "weight": "Bolder",
+                        "text": "Enter Command:",
+                        "isSubtle": true,
+                        "wrap": true,
+                    },
+                    {
+                        "type": "Input.Text",
+                        "id": "commandToBot",
+                        "placeholder": "E.g. timezone",
+                    },
+                ],
+                "actions": [
+                    {
+                        "type": "Action.Submit",
+                        "id": "postCommand",
+                        "title": "Post Command",
+                    },
+                    {
+                        "type": "Action.Submit",
+                        "id": "cancel",
+                        "title": "Cancel",
+                    },
+                ],
+                "version": "1.0",
+            };
+
+             $(document).ready(function () {
+                 microsoftTeams.initialize();
+                 microsoftTeams.registerOnThemeChangeHandler(function(theme) {
+                     document.getElementById('currentTheme').innerHTML = theme;
+                 });
+             });
+             
+             function acAttachment(ac) {
+                return {
+                    contentType: "application/vnd.microsoft.card.adaptive",
+                    content: ac,
+                };
+            }
+
+             document.addEventListener("DOMContentLoaded", function(){
+                let taskModuleButtons = document.getElementsByClassName("taskModuleButton");
+                let taskInfo = {
+                    title: null,
+                    height: null,
+                    width: null,
+                    url: null,
+                    card: null,
+                    fallbackUrl: null,
+                    completionBotId: null,
+                };
+                
+                taskInfo.height = 510;
+                taskInfo.width = 510;
+                let submitHandler = (err, result) => {
+                    document.getElementById('taskModuleResult').innerHTML="you enterd the command: " + result.commandToBot;
```

Comment:
Also, don't use the innerHTML property, as that can make the page susceptible to XSS. Use JQuery's text().